src/Cohesity.Powershell/Scripts/StorageDomain/New-CohesityStorageDomain.ps1

```diff
@@ -0,0 +1,90 @@
+# Request to create a Storage Domain (View Box) configuration.
+#### USAGE ####
+#New-CohesityStorageDomain -Name sample
+#New-CohesityStorageDomain -Name sampleS -Deduplication -InlineDeduplication -InlineCompress -Encryption -ClusterPartitionName DefaultPartition
+###############
+
+function New-CohesityStorageDomain {
+    [CmdletBinding()]
+    Param(
```

Comment:
Its good to add physical quota or logical quota to create a storage domain.

Reply:
Yeah sure, will add those parameters also